en/config/safety.md

```diff
@@ -229,6 +231,21 @@ Parameter | Description
 [VT_FW_ALT_ERR](../advanced_config/parameter_reference.md#VT_FW_ALT_ERR) | Maximum negative altitude error for fixed wing flight. If the altitude drops more than this value below the altitude setpoint the vehicle will transition back to MC mode and enter failsafe RTL.
 
 
+## Failure Detector {#failure_detector}
```

Comment:
@bresch Does failure detector work on all vehicles and modes? What about acro modes, where things are supposed to turn over? How is it disabled for a particular vehicle?

Reply:
Unfortunately, the trigger is active as soon as the circuit-breaker is disabled. It's active for all modes, which means that someone flying in acro will trigger termination during a flip. The plan is to make further improvements after the Firmware PR will be merged (after 1.9 release).

Reply:
Really? That's horrible to the point I'd call it a critical defect against v1.9.

Reply:
Clarification: the circuit breaker is enabled by default; it means that the failure detector is disabled by default. A racer pilot would most likely never disable that circuit breaker, this is why I don't see that as a critical problem.
However, I'll still add a condition such that FD_FAIL_(P|R) = 0 or -1 deactivate the check.

Reply:
Thank you for that clarification. I have updated this to make that clear. Makes a lot more sense now. 

I do find it a little odd that there is one circuit breaker for both attitude and IMU issues